README.md

```diff
@@ -130,6 +130,8 @@ Some of the tests use the prover network to generate the proofs, so you need to
 
 - [Go](https://go.dev/doc/install)
 - [Docker](https://docs.docker.com/get-docker/)
+- [Kurtosis](https://docs.kurtosis.com/install/)
+    - The version of this needs to be the exact same as the `github.com/kurtosis-tech/kurtosis/api/golang` dependency in e2e `go.mod`
```

Comment:
I prefer to put the exact version, v1.8.1. Or you should at least link the `go.mod` file. But I still prefer listing the exact version. I can add this if you don't want to bother